### ISSUE TITLE
PORTUS_BACKGROUND issue when launching Portus container

### ISSUE THREAD

**Issue by mohbar11** [SUSE/Portus#1850]
PORTUS_BACKGROUND="true" causes the error wen lunching a container

When I run this command to start Portus in conainer , I got the error below:
```
docker run -it --rm -p 3000:3000 \
	-v /srv/docker/Portus/certs:/certificates \
	-v /etc/localtime:/etc/localtime:ro \
	-e PORTUS_DB_HOST="172.17.0.2" \
	-e PORTUS_DB_DATABASE="dbname" \
	-e PORTUS_DB_USERNAME="user" \
	-e PORTUS_DB_PASSWORD='password' \
	-e PORTUS_DB_ADAPTER="mysql2" \
	-e PORTUS_MACHINE_FQDN_VALUE="machine.local" \
	-e RAILS_SERVE_STATIC_FILES="true" \
	-e PORTUS_SECRET_KEY_BASE=$(openssl rand -hex 64) \
	-e PORTUS_PASSWORD="1234567890987654321" \
	-e PORTUS_KEY_PATH="/certificates/portus/key.key" \
	-e PORTUS_PUMA_TLS_KEY="/certificates/portus/key.key" \
	-e PORTUS_PUMA_TLS_CERT="/certificates/portus/key.com-ca.crt" \
	-e PORTUS_CHECK_SSL_USAGE_ENABLED="true" \
	-e PORTUS_LDAP_ENABLED="true" \
	-e PORTUS_LDAP_HOSTNAME="ldap.com" \
	-e PORTUS_LDAP_PORT="389" \
	-e PORTUS_LDAP_METHOD="plain" \
	-e PORTUS_LDAP_BASE="ou=People,dc=DC,dc=DC,dc=COM" \
	-e PORTUS_LDAP_UID="uid" \
	-e PORTUS_LDAP_AUTHENTIFICATION_ENABLED="false" \
	-e PORTUS_BACKGROUND=true \
	-e PORTUS_BACKGROUND_REGISTRY_ENABLED="true" \
	-e PORTUS_BACKGROUND_SYNC_ENABLED="true" \
	-e PORTUS_BACKGROUND_SYNC_STRATEGY="update" \
	--name portus1 opensuse/portus:2.3
```

ERROR:

[Mailer config] Host:     machine.local
[Mailer config] Protocol: https://
[Initialization] Running: 'Registry events', 'Registry synchronization'
Exception: Cannot obtain authorization token: #<Net::HTTPUnauthorized:0x0000000006b06980>
Exception: Cannot obtain authorization token: #<Net::HTTPUnauthorized:0x0000000006b62bb8>
Exception: Cannot obtain authorization token: #<Net::HTTPUnauthorized:0x000000000610c7e0>
Exception: Cannot obtain authorization token: #<Net::HTTPUnauthorized:0x00000000060a6968>
Exception: Cannot obtain authorization token: #<Net::HTTPUnauthorized:0x0000000006cb1190>
Exception: Cannot obtain authorization token: #<Net::HTTPUnauthorized:0x0000000006d48400>
Exception: Cannot obtain authorization token: #<Net::HTTPUnauthorized:0x0000000006de5278>

Could you tell what I am doing wrong ? or is there an issue with the PORTUS_BACKGROUND ?
It work fine when I run the docker run without the PORTUS_BACKGROUND env.

**Comment by jpiles** [comment]
Hi, I found the same issue, and it seems related to commit a6f6035d40b9d668352ec4c9c2274b9baa54a748, see https://github.com/SUSE/Portus/commit/a6f6035d40b9d668352ec4c9c2274b9baa54a748#diff-68a7cad80c8fd6efe1c4b9dc0d2934cdL106

The "portus" account is apparently no longer considered special, and if it doesn't exist in the LDAP backend (as is our case, and I guess most people's), it fails, so the backend job can't proceed.

We have reverted to 2.3.2, and so far it seems to work.

**Comment by mssola** [comment]
> The "portus" account is apparently no longer considered special, and if it doesn't exist in the LDAP backend (as is our case, and I guess most people's), it fails, so the backend job can't proceed.

This is not exactly the case, since we simply moved this check into another part of the code, but it looks like it's related to what you're saying. Thanks for the hint! :yum: 

I'll investigate this issue as a bug. Thanks for reporting :+1:

**Comment by ArcticSnowman** [comment]
Can this be fixed in the 2.3 docker image?
